visualisation/visualise-raw-data/update-visualise-raw-data.R

```diff
@@ -0,0 +1,36 @@
+library(tidyverse)
+library(RColorBrewer)
+
+
+source(here::here("visualisation", "visualise-raw-data", 
+                  "visualise-raw-data-functions.R"))
+
+
+current_date <- Sys.Date()
+
+if(!dir.exists(here::here("visualisation", "visualise-raw-data", 
+                          "raw-data-plots", 
+                          current_date))) {
+  dir.create(here::here("visualisation", "visualise-raw-data", 
+                        "raw-data-plots", 
+                        current_date))
+}
+
+
+national_plot <- plot_raw_data()
+
+
+ggsave(here::here("visualisation", "visualise-raw-data", "raw-data-plots", 
```

Comment:
if its in the visualisation folder does it need to have a folder title that is visualise?